### PR TITLE
Add advisory for logging of access tokens in matrix-sdk

### PR DESCRIPTION
**Pull request by jplatte**
Upstream issue: https://github.com/matrix-org/matrix-rust-sdk/issues/1110

### CODE REVIEW

File: crates/matrix-sdk/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "matrix-sdk"
+date = "2022-10-24"
+url = "https://github.com/matrix-org/matrix-rust-sdk/issues/1110"
+
+[versions]
+patched = [">= 0.6.2"]
```
Comment (amousset) [inline crates/matrix-sdk/RUSTSEC-0000-0000.md:9]: shouldn't this be `>= 0.6.1` which [seems to contain the fix](https://github.com/matrix-org/matrix-rust-sdk/releases/tag/matrix-sdk-0.6.1)?

Comment (jplatte) [reply crates/matrix-sdk/RUSTSEC-0000-0000.md:9]: It contains a partial fix only, which I actually only realized after being half way done with the advisory. I'll update the GitHub releases.

Comment (jplatte) [reply crates/matrix-sdk/RUSTSEC-0000-0000.md:9]: Does it make sense like this? https://github.com/matrix-org/matrix-rust-sdk/releases

Comment (amousset) [reply crates/matrix-sdk/RUSTSEC-0000-0000.md:9]: Yes, thanks for the exlanation :+1:

**Comment by amousset** [comment]
Thanks for reporting!

**Pull request merged**